cms/static/cms/js/modules/cms.base.js

```diff
@@ -299,12 +299,7 @@ export const Helpers = {
         });
 
         return newUrl
-            .toString()
-            .split('#')
-            .map((part, i) => {
-                return i === 0 ? part.replace(/&/g, '&amp;') : part;
-            })
-            .join('#');
+            .toString();
```

Comment:
The function documentation needs to be updated to suit the code, it no longer does as this describes:

    /**
     * Modifies the url with new params and sanitises
     * the ampersand within the url for #3404.
     *
     * @method makeURL
     * @param {String} url original url
     * @param {Array[]} [params] array of [`param`, `value`] arrays to update the url
     * @returns {String}
     */

Reply:
Possibly:

/**
 * Modifies the url with new params and sanitises the url.
 *
 * @method makeURL
 * @param {String} url original url
 * @param {Array[]} [params] array of [`param`, `value`] arrays to update the url
 * @returns {String}
 */

Reply:
True! I've also added that any passed `&amp;` are converted to `&`